lib/app/pages/add_folder_words_page/cubit/add_folder_words_cubit.dart

```diff
@@ -7,26 +7,27 @@ import '../../../../domain/use_case/new_data_folder_use_case.dart';
 import '../../../theme/global_imports.dart';
 import '../../../utils/enums/errors.dart';
 
-part 'add_folder_words_state.dart';
-
 part 'add_folder_words_cubit.freezed.dart';
 
+part 'add_folder_words_state.dart';
+
 @injectable
 class AddFolderWordsCubit extends Cubit<AddFolderWordsState> {
   AddFolderWordsCubit(this._createFolderUseCase) : super(const AddFolderWordsState.loaded([]));
 
   final CreateFolderUseCase _createFolderUseCase;
   final List<WordsEntity> wordsList = [];
 
-  Future<void> createFolder({
-    required String folderName,
-  }) async {
+  Future<void> createFolder({String? folderName, String? sourceLanguage, String? targetLanguage}) async {
     emit(const AddFolderWordsState.loading());
     final result = await _createFolderUseCase(
       FolderEntity(
-        folderName: folderName,
+        folderName: folderName ?? '',
```

Comment:
folder name can be null?

Reply:
You can override folder name with an empty string because you will create a folder without name on the database. This can lead to errors